aws/sagemaker_notebook_instance.go

```diff
@@ -0,0 +1,90 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+func getAllNotebookInstances(session *session.Session, excludeAfter time.Time) ([]*string, error) {
```

Comment:
I was wondering if can be implemented filtering by name

https://github.com/gruntwork-io/cloud-nuke/blob/master/aws/s3.go#L257

Reply:
I've added the capability of filtering using the config file!